package.json

```diff
@@ -43,6 +43,6 @@
   },
   "dependencies": {
     "@aws-sdk/util-utf8-browser": "^3.109.0",
-    "aws-crt": "^1.18.0"
```

Comment:
Is there a reason why we're removing the carrot? `^`

Reply:
oups, added it back